packages/shared/meta/selectors.ts

```diff
@@ -33,18 +33,44 @@ export const isMetaConfigurationInitiazed = (store: RootMetaState): boolean => s
 
 export const getPois = (store: RootMetaState): Vector2Component[] => store.meta.config.world?.pois || []
 
-export const getCommsConfig = (store: RootMetaState): CommsConfig => store.meta.config.comms ?? { maxVisiblePeers: DEFAULT_MAX_VISIBLE_PEERS }
+export const getCommsConfig = (store: RootMetaState): CommsConfig =>
+  store.meta.config.comms ?? { maxVisiblePeers: DEFAULT_MAX_VISIBLE_PEERS }
 
 export const getBannedUsers = (store: RootMetaState): BannedUsers => store.meta.config.bannedUsers ?? {}
 
-export const getPickRealmsAlgorithmConfig = (store: RootMetaState): AlgorithmChainConfig | undefined => store.meta.config.pickRealmAlgorithmConfig
+export const getPickRealmsAlgorithmConfig = (store: RootMetaState): AlgorithmChainConfig | undefined =>
+  store.meta.config.pickRealmAlgorithmConfig
 
 export const isMOTDInitialized = (store: RootMetaState): boolean =>
   store.meta.config.world ? store.meta.config.world?.messageOfTheDayInit || false : false
 export const getMessageOfTheDay = (store: RootMetaState): MessageOfTheDayConfig | null =>
   store.meta.config.world ? store.meta.config.world.messageOfTheDay || null : null
 
-export const isFeatureEnabled = (store: RootMetaState, featureName: FeatureFlags, ifNotSet: boolean): boolean => {
+export function getFeatureFlags(store: RootMetaState): Record<string, boolean> {
+  const featureFlags: Record<string, boolean> = {}
```

Comment:
Please use `Record<string, FeatureFlagVariant>` instead of `Record<string, boolean>`

Reply:
Right now I think that kernel is not using the feature flag library, should we implement the library and use it or we can copy the `FeatureFlagVariant ` type?

Reply:
It can be copied, the datastructure in the getStore should be the same